### PR TITLE
Adding test elements to affinity group test

### PR DESCRIPTION
**Pull request by laheara143**
## Describe context / purpose for this PR
*This PR is a WIP*

Feature: test an Affinity Groups page
  An Affinity Group page should display the title, logo, description, and tags.
  Groups that have specified an optional Slack, Q&A, GitHub, or email link should display a button with the appropriate link.
## Issue link
https://cyberteamportal.atlassian.net/browse/D8-1826
## Link to MultiDev instance

http://md-{{ ticket number }}-accessmatch.pantheonsite.io

## Checklist for PR author
- [ ] I have checked that the PR is ready to be merged
- [x] I have reviewed the DIFF and checked that the changes are as expected
- [x] I have assigned myself or someone else to review the PR

### CODE REVIEW

**Comment by laheara143** [comment]
The file that is failing I do not see within my codespace.

**Comment by laheara143** [comment]
Should this affinity-individual also be a test for asp?

**Comment by laheara143** [comment]
How can the CI links be tested within an affinity group page?

**Comment by a-pasquale** [comment]
@laheara143 CI links are webform submissions that are sanitized in the db artifact. We should select an Affinity Group when creating CI Links for testing and then can test on the Affinity Group page as well.

File: tests/behat/features/asp/cssn/cssn.feature [hunk 1]
```diff
@@ -37,7 +37,7 @@ a button "Join the CSSN Network," which links to the login page.
     When I go to "/cssn"
     And I click "FIND OUT MORE"
     And I wait for the page to be loaded
-    Then I should be on "/ccep-pilot"
+    #Then I should be on "/ccep-pilot"
```
Comment (a-pasquale) [inline tests/behat/features/asp/cssn/cssn.feature:40]: This was changed to /ccep

File: tests/behat/features/templates/affinity_groups/affinity_groups-individual.feature [hunk 2]
```diff
@@ -26,34 +51,103 @@ Feature: test individual Affinity Group page
     Then I should see "Cloud Computing"
     Then I should see "cloud-commercial"
     Then I should see "People who use or are"
-    Then I should see "Join"
+#TODO Below line reads Join in menu rather than on th e affinity group page, could we add an id to specify? D8-1830 https://cyberteamportal.atlassian.net/browse/D8-1830
```
Comment (a-pasquale) [inline tests/behat/features/templates/affinity_groups/affinity_groups-individual.feature:54]: when I click the ".btn.affinity-group-flag" element

**Comment by a-pasquale** [comment]
AG tests should be in templates and asp directories. AGs are the same on all domains.

**Comment by laheara143** [comment]
This PR seems to still have ongoing issues with the Join button

**Comment by laheara143** [comment]
@a-pasquale "when I click the ".btn.affinity-group-flag" element" does not seem to be passing anymore

**Comment by laheara143** [comment]
Is there any html element that I can use to test the Allocated CiDeR Resources and Recommended Resources accordions? I tried both "    When I click the ".recommended-resource-header-5945" element" and "    When I click the ".associated-resource-header-388" element" but nothing seems to be working.

**Comment by laheara143** [comment]
Also, I tried figuring out a way to masquerade as a user within behat but since users are sanitized it is more difficult. I will start looking into these links so we can get the last parts tested [Link1](https://www.jeffgeerling.com/blog/2018/logging-existing-user-behat-test-drupal-extension) [Link2](https://drupal.stackexchange.com/questions/278025/behat-javascript-test-can-not-login-user-incorrect-username-or-password-messa)

**Comment by a-pasquale** [comment]
You might try
```when i click the "#associated-resource-388" element```
on /affinity-groups/delta
and 
```when i click the "#recommended-resource-5945" element```
on /affinity-groups/ai-institutes-cyberinfrastructure

**Comment by laheara143** [comment]
> @laheara143 CI links are webform submissions that are sanitized in the db artifact. We should select an Affinity Group when creating CI Links for testing and then can test on the Affinity Group page as well.

When editing an affinity group you can add a ci link

**Comment by a-pasquale** [comment]
> @laheara143 CI links are webform submissions that are sanitized in the db artifact. We should select an Affinity Group when creating CI Links for testing and then can test on the Affinity Group page as well.

Oops, CI Links can be added to an Affinity Group while editing the Affinity Group.

**Comment by laheara143** [comment]
> AG tests should be in templates and asp directories. AGs are the same on all domains.

Asp will be added once templates is passing

**Comment by a-pasquale** [comment]
> This PR seems to still have ongoing issues with the Join button

For anonymous, use .affinity-group-buttons .btn.disabled

**Comment by jasperlieber** [comment]
redone on https://github.com/necyberteam/cyberteam_drupal/pull/856

**Pull request closed without merging**